server-side/email-service/app/producer.js

```diff
@@ -0,0 +1,54 @@
+const kafka = require('kafka-node');
+const Producer = kafka.Producer;
+const Admin = kafka.Admin;
+const client = new kafka.KafkaClient({ kafkaHost: 'kafka:9092' });
+const admin = new Admin(client);
+
+const topicToCreate = [
+  {
+    topic: 'emailTopic',
+    partitions: 1,
+    replicationFactor: 1
+  }
+];
+
+admin.createTopics(topicToCreate, async (err, res) => {
+  if (err) {
+    console.error('Failed to create topics:', err);
+  } else {
+  }
+});
+
+const producer = new Producer(client);
+producer.on('ready', () => {
+  insertEvent('emailTopic', 'sendEmail', {
+    to: 'malkysino@gmail.com',
```

Comment:
remove the hard code, please use parameters

Reply:
you don't need this email try, please remove it